src/components/DownloadAnalysisLink/DownloadAnalysisLink.tsx

```diff
@@ -48,14 +49,18 @@ function onClick(store: Store) {
         percentAdequatePopulation,
         percentInadequatePopulation
       } = summaryStatistics([_], store)
+      let specialty = store.get('providers')[0].specialty // TODO: Is this safe to assume?
+      if (specialty == null) {
+        specialty = 'Undefined'
```

Comment:
Nice fix for the type error, but do we really want to show "Undefined" in the CSV? Currently we show "NaN" which indicates an error, "-" which indicates not implemented, and now "Undefined" for missing data. Should we just leave the cell empty for missing data instead?

Reply:
Switched to '-' for consistency